### ISSUE TITLE
[Azure]Errors within the Bot Channels Registration Analytics page: Errors 500 and 400 

### ISSUE THREAD

**Issue by ItsMeArthur** [Microsoft/BotBuilder#4500]
## Bot Info
* SDK Platform: .NET
* SDK Version: 3.14.1.1
* Active Channels: Skype, MS Teams, Slack, Direct Line
* Deployment Environment: Azure Bot Service (Bot Channels Registration) & Azure App Service

## Issue Description 
The Analytics page within the Bot Channels Registration is presenting several errors for all of my chatbots (20+ chatbots). I've only noticed this now, but last week I was able to visualize all the data in the Analytics page. I've checked the related Application Insights service and all the information like application id, keys and etc are correct.  

## Example

This is an example of how the Analytics page for all of my chatbots is being presented:

![bot_insighs_report_error](https://user-images.githubusercontent.com/11617999/39053918-a403d992-4486-11e8-9f25-dbd1e52a02ad.png)

The grand totals isn't being loaded and the USERS and MESSAGES sections are presenting error messages.

## In Conclusion

Is this something with the Analytics or App Insights of the chatbot applications (maybe in maintenance) or is this a problem with the configuration of my chatbots?

**Comment by austinhuang0131** [comment]
Merged #4501. Original text:

## Bot Info
* SDK Platform: 
* SDK Version: 
* Active Channels: 
* Deployment Environment: Azure Bot Service
## Issue Description
![image](https://user-images.githubusercontent.com/16656689/39055390-1d010e5a-4482-11e8-83f1-d871c2c8dd41.png)

Cannot show users or messages stats. Only the "Retention" is visible.

## Code Example


## Reproduction Steps
1. Enable Application Insight in Azure.
2. Enable analytics in bot channel reg.
3. Go to analytics.

## Expected Behavior
Show stats and the graph.

## Actual Results
See screenshot.

**Comment by arturl** [comment]
@ItsMeArthur, thanks for your report. We have identified the issue and working on a fix. Should be resolved this week.

**Comment by austinhuang0131** [comment]
It seems to be partially resolved. However, the Skype analytics data is missing between April 20 and now.

**Comment by arturl** [comment]
@austinhuang0131, what is your bot id? I'll look into it.

**Comment by austinhuang0131** [comment]
`b02aeeb5-27a7-44a7-9e33-ba79a0a10246`

**Comment by austinhuang0131** [comment]
@arturl Hello, is there a result?

**Comment by arturl** [comment]
@ItsMeArthur Not yet, hang in there.

**Comment by austinhuang0131** [comment]
@arturl I'm @austinhuang0131 not @ItsMeArthur lol... (Yes I know it's confusing)

**Comment by arturl** [comment]
Ahh, too many Arturs on this thread! :-) I know both of you guys are waiting for a fix, I'll report back as soon as I have more info.

**Comment by austinhuang0131** [comment]
Can I get an update on this?

Also the user graph (Not "Retention") is not showing up for me, it's just "loading" all the time despite the message graph already shown.

**Comment by arturl** [comment]
I've opened an internal ticket on the "USERS - LOADING..." issue. 
No update on the user retention yet, I will report on this thread as soon as we have a firm ETA.

**Comment by arturl** [comment]
Quick update: the "USERS - LOADING" issue should be fixed next week, deployment ETA 6/7.
The user retention fix is in the pipeline, though may or may not come out next week.

**Comment by austinhuang0131** [comment]
I said "Not 'Retention'" lol... There's no issue with the retention graph.

What about Skype/Teams?

**Comment by rjgmail88** [comment]
This is closed and not sure why. Quick glance at the end in comments shows it fixed but I still see issue with my Bot #984ede7e-bc35-4767-a19b-60a2d6c5e4be

**Comment by EricDahlvang** [comment]
Hi @rjgmail88 

This issue is not closed.

**Comment by arturl** [comment]
First off, apologies that it has taken us so long too fix. We delivered solution to a partner team that has just deployed it in production, and we verified that the data is flowing again. Thank you for your patience.

**Comment by arturl** [comment]
Error 403 is "forbidden". This could be caused by the access to the bot's AppInsights container. I suggest opening a new bug, providing the bot name, which channel it is connected to, and when did you start seeing the problem (did it ever work for this bot?)

**Comment by rjgmail88** [comment]
@arturl , Yes it work for 4 days when this issue was fixed. Look at my comment above

> Looks like issue is fixed. I am able to see channel specific data.
